lib/workers/branch/index.spec.ts

```diff
@@ -1040,5 +1062,109 @@ describe('workers/branch', () => {
         )
       ).not.toBeUndefined();
     });
+
+    it('executes post-upgrade tasks once when set to branch mode', async () => {
+      const updatedPackageFile: File = {
+        name: 'pom.xml',
+        contents: 'pom.xml file contents',
+      };
+      getUpdated.getUpdatedPackageFiles.mockResolvedValueOnce({
+        updatedPackageFiles: [updatedPackageFile],
+        artifactErrors: [],
+      } as PackageFilesResult);
```

Comment:
Is the cast required? shouldn't it work without?

Reply:
Tried that, but this happens: https://imgur.com/a/78GSEJS

Reply:
It's been a while, but I think I tried removing the `as` for all the lines I updated first, if that failed I'd add it back